src/sphinxstd.h

```diff
@@ -516,6 +516,8 @@ inline int sphLog2 ( uint64_t uValue )
 		return 0;
 	return 64 - __builtin_clzll(uValue);
 #elif _WIN32
+	if (!uValue)
```

Comment:
could you move this check from ALL platform dependent code up function start?

I'd make less copy pasted code.

Reply:
certainly. Moved to function start